specification/metrics/negotiatedcost.md

```diff
@@ -0,0 +1,34 @@
+# Negotiated Cost
+
+The Negotiated Cost represents a charge inclusive of the impacts of volume/tier-based rates and negotiated discounts while excluding any commitment-based discounts or the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Negotiated Cost equals the [List Cost](#listcost) when no negotiated discounts are present. It is used to calculate savings based on negotiation activities, by comparing it with List Cost.
+
+The NegotiatedCost column MUST be present in the billing data. This column MUST be a valid numeric value of type Decimal and MUST NOT contain null values. When a [NegotiatedUnitPrice](#negotiatedunitprice) is not null, multiplying the NegotiatedUnitPrice by [QuantityInPricingUnit](#quantityinpricingunit ) MUST produce the NegotiatedCost. In cases where the NegotiatedUnitPrice is null, the following applies:
+
+* If the line item is based on other charges (e.g. [ChargeType](#chargetype) is 'Tax'), the NegotiatedCost MUST be calculated based on NegotiatedCost of the related charges
+* If the line item is unrelated to other charges (e.g. [AdjustmentType](#adjustmenttype) is 'Credit'), the NegotiatedCost MUST match the BilledCost.
```

Comment:
What about an unused commitment charge?

To me, the negotiated cost should be 0 in this case.

Reply:
If we expect to receive all Purchase and Usage related records, are we supposed to calculate unused commitments or do we also anticipate receiving records for unused commitments?

Reply:
In the Azure amortized costs dataset, I have witness that an unused commitment record exists for every reservation, in which the amortized cost is 0 in case the reservation is used at 100%.
I do not know for GCP and AWS: I believe converters will need to calculate and to generate unused commitment records if they do not exist in GCP and AWS cost and usage data.